src/pages/OrgRegister.tsx

```diff
@@ -1,91 +1,116 @@
-          />
-          <label className="tracking-wide text-gray-700 dark:text-white text-xl font-normal mb-2">
-            Organization Description
-          </label>
-          <div className='absolute px-2 text-gray-400 mt-7 '>
-            <IoCreate className='w-6 h-6' />
-          </div>
-          <textarea
-            className="border border-primary w-full  px-9 py-4 rounded mb-4 dark:border-white dark:bg-[#1F2A37] dark:text-white mt-3"
-            name="Describe"
-          >
-          </textarea>
-          <div className="flex justify-between items-center mb-6">
-            <div className="form-check">
-              <input
-                type="checkbox"
-                className=" h-4 w-4 border border-gray-300 rounded-sm bg-white checked:bg-blue-600 focus:outline-none transition duration-200 mt-1 align-top bg-no-repeat bg-center bg-contain float-left mr-2 cursor-pointer"
-                id="exampleCheck2"
-              />
-              <label className="form-check-label inline-block dark:text-white text-gray-800"
-              >I agree to Pulse's <span className='text-primary'>Terms & Conditions</span>
-              </label
-              >
-            </div>
+    <div className="md:flex md:flex-col md:items-center md:justify-center w-full  min-h-screen  text-center py-2 -mb-[10%] -mt-[4%] dark:bg-dark-bg bg-gray-100  sm:flex sm:flex-row sm:items-center sm:justify-center">
+      <div className="bg-white md:rounded-2xl md:shadow-2xl md:flex 2xl:w-2/3 md:w-[90%] mt-6 md:mt-24 md:mb-24 xl:mt-16 xl:mb-0 md:max-w-4xl sm:max-w-2xl sm:rounded-none sm:shadow-none dark:shadow-2xl ">
+        <div className="2xl:w-3/5 md:w-screen 2xl:p-5 md:p-14 sm:w-full sm:p-2 dark:bg-dark-frame-bg dark:rounded-tl-xl dark:rounded-bl-xl  ">
+          <div className="py-10 sm:py-4 ">
+            <h2 className="text-2xl pb-6 font-bold text-primary dark:text-dark-text-fill ">
+              Register your organization
+            </h2>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <div className="flex flex-col items-center">
+                <label className="tracking-wide text-gray-700 pr-64 md:pr-48 lg:pr-80 dark:text-white font-normal">
+                  E-mail
+                </label>
+                <div className="rounded border border-primary dark:border-white w-full md:w-3/4 p-2 flex items-center mb-6 dark:bg-dark-bg ">
+                  <FaRegEnvelope className="text-gray-400 mr-2" />
+                  <input
+                    type="email"
+                    {...register("email", { required: "email is required" })}
+                    className="bg-white outline-none text-sm flex-1 dark:border-white dark:bg-dark-bg dark:text-white"
+                  />
+                </div>
+                <div className='-mt-6 pr-[65%] md:pr-[45%] lg:pr-[55%]'>
+                  {errors.email && (<small className='text-red-600'>email is required</small>)}
```

Comment:
Also retrieve the error from the error object provided by the react-hook-form instead of hard-coding the value